pkg/tracker/currentVariables.go

```diff
@@ -49,43 +47,43 @@ func (b *CurrentVariablesTracker) Exec(ctx context.Context) error {
 	// if we've mined it, don't save it.
 	myStatus, err := instance.DidMine(nil, currentChallenge, fromAddress)
 	if err != nil {
-		fmt.Println("My Status Retrieval Error")
+		level.Error(logger).Log("msg", "My Status Retrieval Error", "err", err)
 		return err
 	}
 	bitSetVar := []byte{0}
 	if myStatus {
 		bitSetVar = []byte{1}
 	}
-	currentVarsLog.Info("Retrieved variables. challengeHash: %x", currentChallenge)
+	level.Info(logger).Log("msg", "Retrieved variables", "challengeHash", currentChallenge)
 
 	err = DB.Put(db.CurrentChallengeKey, currentChallenge[:])
 	if err != nil {
-		fmt.Println("Current Variables Put Error")
+		level.Error(logger).Log("msg", "Current Variables Put Error", "var", "currentChallenge", "err", err)
 		return err
```

Comment:
I think we should simplify this and the following lines and just return an error here and then the caller can print the log.
```suggestion
		return errors.Wrap(err,"currentChallenge variables put")
```